### ISSUE TITLE
Formulae

### ISSUE THREAD

**Issue by Ulvtand** [AnthonyMonette/TheWitcherTRPG#102]
Add checkbox "Formula" to all Items

**Comment by AnthonyMonette** [comment]
I don't think this is necessary there is already an item of type diagram that can be use to descrbe a formulae.

**Comment by Ulvtand** [comment]
Necessary, no not on the whole but it would be much easier to duplicate an object and "just" fill in what is required for the formulas.
:)
